pom.xml

```diff
@@ -45,7 +45,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <raml-module-builder.version>34.0.0</raml-module-builder.version>
+    <raml-module-builder.version>35.0.0-SNAPSHOT</raml-module-builder.version>
```

Comment:
I'm a little confused. That issue seems to refer to about the ability to provide multiple set-cookie headers. And is closed and included in RMB 35.x

I thought a previous comment on this issue suggested that wasn't fixable and we needed a permanent workaround.

What have I missed?

Reply:
In effect, there are two workaround, one in RMB to allow for multiple headers and another in this module to use them, because the generated code does not support that.

Reply:
I think the workaround in RMB to just honor headers is a fix.

Reply:
> I think the workaround in RMB to just honor headers is a fix.

Sure, there is a fix (in RMB to support multiple headers) and a workaround (to access that by not using the generated responses, because folks were meant to use those generated responses)